android/expoview/build.gradle

```diff
@@ -160,9 +162,42 @@ dependencies {
       modulesPaths : ['../../packages'],
       configuration: 'api',
       target       : 'react-native',
-      exclude      : ['expo-bluetooth', 'expo-in-app-purchases', 'expo-updates']
+      exclude      : [
+          'expo-bluetooth',
+          'expo-in-app-purchases',
+          'expo-updates',
+          'unimodules-react-native-adapter',
+          'expo-branch'
+      ]
   ])
 
+  // 1. React Native 0.60+ required 3rd party modules libraries to depend on it
+  //    via `api` or `implementation` configurations.
+  //    See https://github.com/unimodules/react-native-unimodules/issues/67#issuecomment-529892115
+  //        https://github.com/expo/expo/pull/5648/commits/29a3a4f29c0410c7824d004a85fa836e3db5280c
+  // 2. Just this simple change to unimodules' `build.gradle`s fixed the issue
+  //    in developers' projects, but rendered Expo client project non-buildable:
+  //    `ReactAndroid` and `com.facebook.react:react-native` were in conflict,
```

Comment:
Is there some way we could tell Gradle to use ReactAndroid whenever it tries to resolve `com.facebook.react:react-native`?

Reply:
Let's find out! https://github.com/expo/expo/pull/5695/commits/e42e4fc36a5f23563046647f86710763038bcc88

Reply:
That did it! https://github.com/expo/expo/pull/5695/files/99df0d98e220d3f1b185a5a874c39a72d1939816..724798cd36ae41cc6020baa93924c8a75335073b